spec/rubocop/cop/style/double_cop_disable_directive_spec.rb

```diff
@@ -3,7 +3,7 @@
 RSpec.describe RuboCop::Cop::Style::DoubleCopDisableDirective do
   subject(:cop) { described_class.new }
 
-  it 'registers an offense when using `#bad_method`' do
+  it 'registers an offense when using `#bad_method` with disable' do
```

Comment:
It seems I was the one who named this example originally, but now it makes no sense to me. I don't know what `#bad_method` or `#good_method`means. :slightly_smiling_face:  Could you come up with something better?

Reply:
Addressed :)